docs/checks.md

```diff
@@ -73,10 +73,15 @@ result to meet most user needs.
 
 Different types of branch protection protect against different risks:
 
-  - Require code review: requires at least one reviewer, which greatly
-    reduces the risk that a compromised contributor can inject malicious code.
-    Review also increases the likelihood that an unintentional vulnerability in
-    a contribution will be detected and fixed before the change is accepted.
+  - Require code review: 
```

Comment:
Changes LGTM but please add them to checks.yaml instead of checks.md, which is autogenerated. (Run `make generate docs`)

Reply:
Thanks @olivekl. I've changed to the right file!

Reply:
I've generated the checks.md using `make generate-docs` also